### PR TITLE
Fix: Resolve the issue where the statement import "file" fails to loa…

### PR DESCRIPTION
**Pull request by wuchuheng**
Fix: Resolve the issue where the statement import "file" fails to load file.js unless the .js suffix is explicitly included

Screen Shot 2023-12-12 at 15 56 40

### CODE REVIEW

**Comment by hsiaosiyuan0** [comment]
Maybe is not a fix since it's qjs default behavior to resolve module name, if you'd like to add this feature, please do it as a secondary manner, which means we should keep the default behavior exists and switch to your manner only if the module cannot be found via the default behavior.

**Comment by wuchuheng** [comment]
First of all I totally agree with you, and your concerns. And that is indeed what I did. Enabling this option for myself only after the original code didn't find that module. So doing that, what is the point of the code? What problem did the  codes  actually solve in js? It's because when I wrote the ts codes,  when the ts codes converted to JS codes, the js  codes
 didn't add a js suffix, which resulted in the code compiled from ts to js not being able to run with qjs. So in order to solve this problem, so a PR .

Before that, I also tried to solve this issue

 without modifying the source code of the qjs , but there is no exception, too much trouble, it is better to modify the source code directly .


  To summarize one reason for doing this is that it makes no sense for me to modify my quick js to work when my code written in ts works just fine in nodejs.

That's the reason.

以上！
